### DISCUSSION TITLE
如何下载element-plus离线版文档

### DISCUSSION THREAD

**Discussion by liupengfei12** [element-plus/element-plus#6425] [Q&A]
请问如何下载离线版文档？我把代码拉到本地，pnpm install 以后pnpm run build 打包的好像是模块包没有index.html。在docs里面执行同样的操作打包完本地因为同源策略不能使用，我对这方面不太了解，请问如何正确打包

**Comment by sxzz** [comment]
https://github.com/element-plus/element-plus/issues/6182#issuecomment-1046256082

**Reply by liupengfei12** [reply]
对不起 我按照方法下载了以后index..html里面引入的js和css跨域，我把绝对路径改为相对路径就会显示页面不存在，我是菜鸡

**Comment by tolking** [comment]
现在文档已经支持PWA。

理论上如果你的浏览器支持，在访问 [https://element-plus.org/zh-CN/](https://element-plus.org/zh-CN/) 后经过几分钟的缓存就可以离线访问了（具体进度查看调试中的网络面板）。

如果是 Chrome 在地址栏右侧会出现一个安装应用图标，也可以选择安装作为 Web App 启动
